src/components/MenuItem/MenuItem.component.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import {Icon} from '@Components/Icon';
+
+import {MenuItemProps} from './MenuItem';
+import {Label, MenuItemWrapper, SecondaryLabel} from './MenuItem.styles';
+
+/**
+ * MenuItem Component
+ *
+ * @example Correct usage
+ *
+ * ```ts
+ * <MenuItem label='Menu Item Label' />
+ * ```
+ *
+ * @example usage with secondary label positioned below main label
+ * ```ts
+ * <MenuItem label='Main Label'
+ *           secondaryLabel='Secondary Label'
+ * />
+ * ```
+ *
+ * @example usage with secondary label positioned inline with main label
```

Comment:
```suggestion
 * @example usage with secondary label positioned inline with the main label
```
use can use the Grammarly chrome extension for checking grammars